File: Assignment No_27/Question_4.c
/*4. Write a program which accept string from user and check whether it contains vowels in it or not.

Input : "marvellous"
Output : TRUE

Input : "Demo"
Output : TRUE

Input : "xyz"
Output : FALSE
 

*/
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int Check_vowels(char *str)
{
    while (*str != '\0')
    {
        if (*str == 'A' || *str == 'E' || *str == 'I' || *str == 'O' || *str == 'U' || *str == 'a' || *str == 'e' || *str == 'i' || *str == 'o' || *str == 'u')
            return 1;
        str++;
    }
    return 0;
}


int main()
{
   char Arr[20];
   int iRet=0;

   printf("Enter the string:\n");
   scanf("%[^'\n']s",Arr);

   iRet=Check_vowels(Arr);

   if (iRet == 1)
        printf("TRUE\n");
   else
        printf("FALSE\n");

   return 0;
}

File: Assignment No_36/Question_4.java
/* 4. Write a java program which accept string from user and check whether it contains vowels in it or not.

Input : “marvellous”
Output : TRUE

Input : “Demo”
Output : TRUE

Input : “xyz”
Output : FALSE
*/
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.*;

class Ass36_4
{
    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter The String : ");
        String name = sobj.nextLine();

        int i = 0, iAns=0;
        char ch;

        for(i = 0; i < name.length(); i++)
        {
            ch = name.charAt(i);

            if((ch == 'a') || (ch == 'e') || (ch == 'i') || (ch == 'o') || (ch == 'u'))
            {
                iAns++;
                break;
            } 
        }        

        if(iAns > 0)
        {
            System.out.println("TRUE");
        }
        else 
         {
            System.out.println("FALSE");
         }   

    }
}
